common/scala/src/main/resources/application.conf

```diff
@@ -352,6 +352,7 @@ whisk {
     activation {
         payload {
             max = 1048576
+            truncation = 1048576
```

Comment:
nit: I think it is easier to read if these are (both) changed to "m" units (mebibytes per hocon), e.g.
```
            max = 1 m
            truncation = 1 m
```